cdap-ui/app/features/dashboard/controllers/widget.js

```diff
@@ -136,7 +162,6 @@ angular.module(PKG.name+'.feature.dashboard')
     });
     $scope.wdgt.fetchData($scope);
     $scope.chartHistory = null;
-    $scope.stream = null;
```

Comment:
Any specific reason why we are removing this? Its generally a better practice to declare the variable inside a scope before assigning with values from backend/services